roles/openshift_manage_node/defaults/main.yml

```diff
@@ -4,3 +4,6 @@ openshift_manage_node_is_master: False
 
 # Default is to be schedulable except for master nodes.
 l_openshift_manage_schedulable: "{{ openshift_schedulable | default(not openshift_manage_node_is_master) }}"
+
+openshift_master_node_label:
+  node-role.kubernetes.io/master: true
```

Comment:
Does capitalization matter here, and do we need for this to be a string?  I'm unsure how this works, just would like to make sure.

Reply:
Its created with capital T - I've updated the PR to ensure it matches 'node-role.kubernetes.io/master=true' (`oc get nodes -o wide` showed correct role however)

Making it a string is possible, although in the end it would be converted in a dict for `oc_label`, so defining it as a dict in YAML is easier